README.md

```diff
@@ -1,118 +1,7 @@
-## Eclipse Che documentation
+# Eclipse Che documentation project
 
-Che docs use Jekyll to convert `.adoc` (AsciiDoc) files into HTML pages. Docs are published at [https://www.eclipse.org/che/docs](https://www.eclipse.org/che/docs/). Updates are synced with a release cycle.
+This repository contains the documentation for the Eclipse Che. See your [Contribution guide](CONTRIBUTION.md) to learn how to contribute. We love pull requests and appreciate contributions that make docs more helpful for users.
 
-## Build and preview
+To ask a question, [file an issue](https://github.com/eclipse/che/issues/new?labels=area/doc,kind/question).
```

Comment:
Maybe emphasize that `eclipse/che/issues` is not a place for support questions? Unless it is.

Reply:
I don't understand. Where else are users expected to ask questions?

Reply:
I believe it is. There 'question' label for it.